lms/djangoapps/certificates/admin.py

```diff
@@ -28,6 +29,11 @@ def __init__(self, *args, **kwargs):
         self.fields['organization_id'] = forms.TypedChoiceField(
             choices=org_choices, required=False, coerce=int, empty_value=None
         )
+        lang_choices = all_languages()
+        lang_choices.insert(0, ('', '------------'))
```

Comment:
What happens if the user saves the form without selecting a language. Does `''` get inserted into the DB, or `null`? We should make sure we're consistent about how we save a template without a language.